packages/react/src/themes/types.ts

```diff
@@ -142,6 +183,14 @@ export type ColorValues<T, Colors extends string | number | symbol = ColorNames>
   T
 >
 
+/**
+ * A type for an extendable set of CategoryColorNames properties of type T
+ */
+type CategoryColorValues<
```

Comment:
`ColorValues` is already a generic, can't we just reuse it with different params